### ISSUE TITLE
S4055 FP: Localizable attribute with false argument in constructor

### ISSUE THREAD

**Issue by csaba-sagi-sonarsource** [SonarSource/sonar-dotnet#4502]
### Description

Rule S4055 should not raise an issue if parameters or properties have [Localizable(false)] attribute.

### Repro steps

```
public void DoWork()
{
    LogMessage(
        "Debug", // This should *not* raise S4055
        "My debug message" /* This is fine if it warns */);
}

public void LogMessage(
    [Localizable(false)]string messageType, 
    string actualMessage)
{
    switch (messageType)
    {
        case "Debug":
            // handle debug
            break;
        case "Trace":
            // etc.
            break;
    }
}
```

### Expected behavior

No issues are raised if parameters or properties have [Localizable(false)] attribute.

### Actual behavior

Issues are raised if parameters or properties have [Localizable(false)] attribute.

### Known workarounds

-

### Related information

* C#/VB.NET Plugins version : 8.23
